rules/consistent-sharepoint-sites/rule.md

```diff
@@ -0,0 +1,24 @@
+---
+type: rule
+title: Do you have your SharePoint sites consistent?
+uri: consistent-sharepoint-sites
+authors:
+  - title: Warwick Leahy
+    url: warwick-leahy
+  - title: Jean Thirion
+    url: jean-thirion
+created: 2022-05-05T04:49:43.291Z
+guid: 55742b01-36da-491a-84d8-5855fee4f2bd
+---
+It's important that all of your SharePoint Sites are as consistent as possible.\
+\
+This helps the user to navigate a new page easily.  They need to know exactly where to look.
+
+<!--endintro-->
+
+Following some simple rules makes this really easy.  
+
+1. Always put any navigation in the same place
+2. Use the same icons for the same types of things, so that a link to "Home" looks the same on each page. Or a user knows when they click on a link with a Word icon, a Microsoft Word document is going to open.
+
+   ![Figure: Keep your SharePoint pages looking consistent](consistent.png)
```

Comment:
Change Figure from: Keep your SharePoint pages looking consistent 
To: Keep pages look consistent - notice the common navigation elements in the same spot